payjoin/src/receive/v2.rs

```diff
@@ -238,18 +238,23 @@ impl Enrolled {
         }
     }
 
+    pub fn pj_uri_builder(&self, address: Address) -> Result<PjUriBuilder, Error> {
```

Comment:
does it make sense for this to be fallable if this state is already `Enrolled`, proving the pj_url correct? Shouldn't enrolling fail if no pj_url is available?